crates/uv/src/commands/python/install.rs

```diff
@@ -385,18 +444,23 @@ pub(crate) async fn install(
             .expect("We should have a bin directory with preview enabled")
             .as_path();
 
+        let upgradeable = preview.is_enabled() && is_default_install
+            || requested_minor_versions.contains(&installation.key().version().python_version());
+
         create_bin_links(
             installation,
             bin,
             reinstall,
             force,
             default,
+            upgradeable,
             is_default_install,
             first_request,
             &existing_installations,
             &installations,
             &mut changelog,
             &mut errors,
+            preview,
         )?;
```

Comment:
I'm finding it confusing that we're passing `preview` in here _and_ using `preview` to determine if the value of `upgradable`?